PocketKit/Sources/PocketKit/FeatureFlagService.swift

```diff
@@ -60,12 +60,15 @@ class FeatureFlagService: FeatureFlagServiceProtocol {
 /// Describes the current feature flags that iOS cares about
 public enum CurrentFeatureFlags: String, CaseIterable {
     case listen = "temp.ios.listen"
+    case listenTagsPlaylists = "temp.ios.listen.tag_playlists"
 
     /// Description to use in a debug menu
     var description: String {
         switch self {
-        case.listen:
+        case .listen:
             return "Enable the Listen feature"
+        case .listenTagsPlaylists:
```

Comment:
I'm missing something; is the debug menu something that's coming, or something that exists? If that's the case, can we create a protocol or something that adds a `debugDescription` to make it more obvious that the description is for debugging purposes? Just an idea, not a blocker.